lib/membrane_http_adaptive_stream/source.ex

```diff
@@ -250,6 +252,19 @@ defmodule Membrane.HTTPAdaptiveStream.Source do
     end)
   end
 
+  defp get_discontinuity_events(%{initial_discontinuity_event_sent?: false} = state) do
+    how_much_skipped = ClientGenServer.how_much_skipped(state.client_genserver)
+
+    get_pads(state)
+    |> Enum.flat_map(fn pad_ref ->
+      [event: {pad_ref, %Membrane.Event.Discontinuity{duration: how_much_skipped}}]
+    end)
+  end
+
+  defp get_discontinuity_events(_state) do
+    []
+  end
```

Comment:
discontinuity events are supposed to be sent up to once on every pad, am I right?